antora.yml

```
@@ -18,9 +18,13 @@ asciidoc:
     experimental: ""
     favicon: favicon.png
     icons: font
+    ocp: OpenShift&#160;Container&#160;Platform
+    # recommended OCP version for new deployments
+    # included in links to OCP docs and in catalog source links: do not include the "v" prefix here
+    ocp4-ver: "4.12"
     # for the project
     che-plugin-registry-directory: che-plugin-registry
-    devworkspace-operator-index: registry.redhat.io/redhat/redhat-operator-index:v4.10
+    devworkspace-operator-index: registry.redhat.io/redhat/redhat-operator-index:v{ocp4-ver}
     devworkspace-operator-version-patch: "0.19.1"
```

I'm checking the Cabal call doc, and it reads `0.20`, although I'm not sure what's the correct value in the X.Y.Z format, like this?
```suggestion
    devworkspace-operator-version-patch: "0.20.0"
```

---

3.5 uses 0.19; 3.6 uses 0.20. I wouldn't change this until you're updating the 3.6 docs.

---

@nickboldt, thanks, we already branched out to a separate branch for 3.5. So I'm committing this suggestion, OK?

---

yes, the correct version is 
```
devworkspace-operator.v0.20.0
```